src/QAT/qat/purr/compiler/emitter.py

```diff
@@ -72,17 +73,6 @@ def add_meta(self, instruction):
     def instructions(self):
         return [seg.instruction for seg in self.timeline]
 
-    def get_pp_for_variable(self, target_var):
```

Comment:
Spirit of the change is fine, but I'd recommend you make the field private, put this method back in, it uses the dictionary and you also throw a meaningful error if the key dosen't exist.

It never failed before because if it found nothing it'd just return an empty dictionary.

Reply:
good point, `get_pp_for_variable` could be used by customers and this shouldn't break that, will push changes shortly.